README.adoc

```diff
@@ -66,17 +66,18 @@ To try the application, navigate to the `finish` directory and run the following
 mvn install liberty:start-server
 ```
 
-Maven builds the application and runs it inside of Open Liberty.
+Maven builds the application and runs it inside Open Liberty.
 
-Point your browser to the `http://localhost:9080/inventory/systems/` URL. Because you recently started the application,
-hosts are not currently registered. Point to the `http://localhost:9080/inventory/systems/localhost` URL to add a host.
+Point your browser to the `http://localhost:9080/inventory/systems` URL. 
+Because you just started the application, the inventory is currently empty. Point to the 
+`http://localhost:9080/inventory/systems/localhost` URL to add the localhost host into the inventory.
 
-Next, point to the `https://localhost:9443/metrics` MicroProfile Metrics endpoint. Click the `Advanced` button to
-proceed. Use the `confAdmin` and `microprofile` credentials to log in. You can see both the standard metrics and
-the application metrics in a text format.
+Next, point to the `https://localhost:9443/metrics` MicroProfile Metrics URL. Click the `Advanced` 
+button to proceed. Use the `confAdmin` and `microprofile` credentials to log in. You can see both
+the standard metrics and the application metrics in a text format.
 
-To see only the application metrics, point your browser to
-`https://localhost:9443/metrics/application`:
+ To see only the application metrics, point your browser to `https://localhost:9443/metrics/application`. 
```

Comment:
We have an extra space in front of "To" that I think we can remove.

Reply:
Fixed